<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/qss.qrc">
    <normaloff>:/ui/image/icon/log.png</normaloff>:/ui/image/icon/log.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>230</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hla_title">
    <item>
     <widget class="QLabel" name="label_log">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10000</width>
        <height>100</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_name">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Matrix</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vla_actAndPwd">
    <item>
     <widget class="QLineEdit" name="lineEdit_account">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>126</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hla_quitAndMin">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_minimize">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_shutdown">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hla_rpwAndAlogin">
    <item>
     <widget class="QCheckBox" name="cbx_remPw">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>记住密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbx_autoLogin">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>自动登录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/qss.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_minimize</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_shutdown</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
